core/src/components/button/button.ionic.scss

```diff
@@ -11,12 +11,11 @@
   --focus-ring-color: #{globals.$ion-border-focus-default};
   --focus-ring-width: #{globals.$ion-border-size-050};
 
-  position: relative;
+  @include globals.typography(globals.$ion-body-action-md);
 
-  min-height: globals.$ion-scale-1000;
+  position: relative;
 
-  font-family: globals.$ion-font-family;
-  font-size: globals.$ion-font-size-350;
+  min-height: globals.$ion-scale-1200;
```

Comment:
Can we move the `min-height` and any specific size styles out to `:host(.button-medium)` so that if we decide to change the default in the future we can just switch the property default?